web/src/js/ducks/view.js

```diff
@@ -0,0 +1,31 @@
+import { SELECT_FLOW } from './flows'
+const ACTIVE_MENU = 'ACTIVE_MENU'
+
+
+const defaultState = {
+    active_menu: 'Start'
+}
+export default function reducer(state = defaultState, action) {
+    switch (action.type) {
+        case ACTIVE_MENU:
+            return {
+                ...state,
+                active_menu: action.active_menu
+            }
+        case SELECT_FLOW:
+            return{
+                ...state,
+                active_menu: action.flowId ? 'Flow' : 'Start'
```

Comment:
So, we probably don't want to change the tab when we switch between flows. For example, when I select a flow, then select the start tab, then press arrow up to select the flow above, I don't want to have the "Flow" tab again.
Unfortunately, this brings up the problem that we'd actually need the previous flow state here, but we only have the view state. This is tricky: http://redux.js.org/docs/FAQ.html#reducers-share-state

Possible workarounds:
1. Custom rootReducer
2. Transfer previous state as part of the action (we can access the full state in the actionCreater using redux-thunk's `getState`.
3. Keep the UI state in React and use `componentWillReceiveProps` (http://redux.js.org/docs/FAQ.html#organizing-state-only-redux-state)

My gut feeling is that 2 or 3 should be the way to go. Thoughts?